### PR TITLE
duf: add page

### PR DESCRIPTION
**Pull request by walking-octopus**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/duf.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`duf`
+
+- List everything (including pseudo, duplicate, inaccessible file systems):
+
+`duf --all`
+
+- Only list specific devices & mount points:
+
+`duf /home /some/file`
```
Comment (einverne) [inline pages/common/duf.md:16]:
```suggestion
`duf {{/path/to/directory1 /path/to/directory2}}`
```

File: pages/common/duf.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- List accessible devices:
+
+`duf`
+
+- List everything (including pseudo, duplicate, inaccessible file systems):
+
+`duf --all`
+
+- Only list specific devices & mount points:
```
Comment (einverne) [inline pages/common/duf.md:14]:
```suggestion
- Only list specific devices or mount points:
```

File: pages/common/duf.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`duf /home /some/file`
+
+- Sort the output:
```
Comment (einverne) [inline pages/common/duf.md:18]:
```suggestion
- Sort the output by size:
```

File: pages/common/duf.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`duf --sort size`
```
Comment (einverne) [inline pages/common/duf.md:20]:
```suggestion
`duf --sort {{size}}`
```

Are there any other options we can use here except size?

**Review by einverne — CHANGES_REQUESTED** [review]
Hi @walking-octopus , thank you for this new page, I left some comments for you to take a look.

File: pages/common/duf.md (hunk 4 above)
Comment (walking-octopus) [reply pages/common/duf.md:20]: > Valid keys are: mountpoint, size, used, avail, usage, inodes, inodes_used, inodes_avail, inodes_usage, type, filesystem.

Comment (walking-octopus) [reply pages/common/duf.md]: Thanks. I've added it in the last commit.

**Comment by walking-octopus** [comment]
Should I add filtering options to the page? Which ones are useful enough to mention?

[DUF filtering options](https://github.com/muesli/duf#filtering=)

**Review by einverne — APPROVED** [review]

**Comment by einverne** [comment]
> Should I add filtering options to the page? Which ones are useful enough to mention?
> 
> [DUF filtering options](https://github.com/muesli/duf#filtering)

I'm not quite familiar with `duf`, let me try. You can add filter options if you think it is worth to be noticed.

File: pages/common/duf.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`duf`
+
+- List everything (including pseudo, duplicate, inaccessible file systems):
+
+`duf --all`
+
+- Only list specific devices or mount points:
+
+`duf {{/path/to/directory1 /path/to/directory2}}`
```
Comment (EmilySeville7cfg) [inline pages/common/duf.md:16]:
```suggestion
`duf /{{path/to/directory1 path/to/directory2 ...}}`
```

If leading slash is mandatory you can put it before `{{`.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by walking-octopus** [comment]
Oops. I accidentally the commit `duf: add filter options`, instead of `duf: fix formatting`.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by CleanMachine1** [comment]
> Oops. I accidentally write `duf: add filter options`, instead of `duf: fix formatting`.


No problem. When we merge PRs, all commits are squished into one to keep a clean history

File: pages/common/duf.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# duf
+
+> Disk Usage/Free Utility - a better 'df' alternative.
```
Comment (CleanMachine1) [inline pages/common/duf.md:3]:
```suggestion
> Disk Usage/Free Utility.
```

File: pages/common/duf.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/muesli/duf>.
+
+- List accessible devices:
+
+`duf`
+
+- List everything (including pseudo, duplicate, inaccessible file systems):
```
Comment (CleanMachine1) [inline pages/common/duf.md:10]:
```suggestion
- List everything (such as pseudo, duplicate or inaccessible file systems):
```

File: pages/common/duf.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`duf --all`
+
+- Only list specific devices or mount points:
```
Comment (CleanMachine1) [inline pages/common/duf.md:14]:
```suggestion
- Only show specified devices or mount points:
```

File: pages/common/duf.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`duf /{{path/to/directory1 path/to/directory2 ...}}`
```
Comment (CleanMachine1) [inline pages/common/duf.md:16]:
```suggestion
`duf {{path/to/directory1 path/to/directory2 ...}}`
```

File: pages/common/duf.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+- List everything (such as pseudo, duplicate or inaccessible file systems):
+
+`duf --all`
+
+- Only show specified devices or mount points:
+
+`duf {{path/to/directory1 path/to/directory2 ...}}`
+
+- Sort the output:
```
Comment (CleanMachine1) [inline pages/common/duf.md:18]:
```suggestion
- Sort the output by a specified criteria:
```

**Review by CleanMachine1 — APPROVED** [review]
I feel this reads better but feel free to ignore this change

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**